code/modules/surgery/organs/organ.dm

```diff
@@ -224,7 +221,7 @@
 /obj/item/organ/proc/receive_damage(amount, silent = 0)
 	if(tough)
 		return
-	if(status & ORGAN_ROBOT)
+	if(is_robotic())
```

Comment:
Below is the hidden buff that internal organs get for being robotic...for no reason.